include/fem-dem/ib_particles_dem.h

```diff
@@ -124,8 +127,62 @@ class IBParticlesDEM
                              std::vector<Tensor<1, 3>> &contact_force,
                              std::vector<Tensor<1, 3>> &contact_torque);
 
+
+  /**
+   *  @brief Calculates particle-particle lubrication force. The force is based on the formula from
+   *  Microhydrodynamics: Principles and Selected Applications by Kim, Sangtae;
+   * Karrila, Seppo J. ISBN 13: 9780750691734
+   *
+   * @param dt_dem The sub time stepping time step.
+   *
+   * @param h_max The gap between particle and the wall below which we evaluate the force.
```

Comment:
```suggestion
   * @param h_max The gap between the particle and the wall below which we evaluate the force.
```

Reply:
Also, shouldn't it be gap between two particles and not particle wall? Since it is calculate_pp_lubrication force